client/hosting/server-settings/components/cache-card/index.tsx

```diff
@@ -75,46 +60,17 @@ export const CacheCard = ( {
 
 	const isClearingCache = isClearingWordpressCache || clearEdgeCacheLoading;
 
-	const clearCache = () => {
+	const handleClearCache = () => {
 		if ( isEdgeCacheActive ) {
 			recordTracksEvent( 'calypso_hosting_configuration_clear_wordpress_cache', {
 				site_id: siteId,
 			} );
+			// @ts-expect-error TODO: Fix this
 			clearEdgeCache();
```

Comment:
I have rewritten the component to TSX.
But here we have an error. To avoid making extra effort for this task, I temporarily muted the error.